manifests/07-operator.yaml

```diff
@@ -4,7 +4,7 @@ kind: Deployment
 metadata:
   name: cluster-image-registry-operator
   namespace: openshift-image-registry
-  annotations:
+  labels:
     config.openshift.io/inject-proxy: operator 
```

Comment:
but this value is wrong.  it's supposed to be the name of the container you want injected.  your container is named cluster-image-registry-operator

Reply:
Updated.  Interesting ... Can you point me to some documentation on that?  I looked all over when I set this up and only found another operator that was doing it and theirs just said operator like this one did.

Reply:
@coreydaley https://github.com/openshift/cluster-version-operator/pull/224#issue-298946446

i dunno that it's documented anywhere unfortunately. 

@danehans we need to make sure teams that have added this label to their operator deployment did so correctly.  the value of the label needs to be the name(s) of the container(s) that are to get the env var injection.